assets/src/blocks/Counter/CounterFrontend.js

```diff
@@ -44,6 +39,18 @@ export class CounterFrontend extends Component {
     if (target !== prevTarget || completed !== prevCompleted || completed_api !== prevCompletedApi) {
       this.calculateRemaining();
     }
+    this.appendENForm();
+  }
+
+  appendENForm() {
+    // Append the counter inside the En-Form
+    var counterBar = document.querySelector('.wp-block-planet4-blocks-counter .counter-style-en-forms-bar')?.cloneNode(true);
```

Comment:
What's the use of `?`?

Reply:
It's the [optional chaining operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining). Here if the element doesn't exist, that line would throw a `TypeError: document.querySelector(...) is null`, but if I use the optional chaining it will just return `undefined`, so I don't have to do a preliminary check on this value.